pkg/services/accesscontrol/roles.go

```diff
@@ -16,8 +17,9 @@ var ldapAdminReadRole = RoleDTO{
 }
 
 var ldapAdminEditRole = RoleDTO{
-	Name:    ldapAdminEdit,
-	Version: 1,
+	Name:        ldapAdminEdit,
+	Version:     2,
+	Description: "Allows every read action for LDAP and in addition allows to administer LDAP.",
```

Comment:
```suggestion
	Description: "Allows every read action for LDAP and in addition allows to administer LDAP.",
```
Allows every read action [to do what?] ... allows [who?]

Reply:
This role seem to inherit its permission list from the ldapAdminReadRole role (so ActionLdapUsersRead and ActionLDAPStatusRead) and on top of that there is a new permission appended to the list: ActionLDAPUsersSync. I'm assuming this protect the HTTP endpoint to retrieve/update users from an ldap DIT and remove users when they are not in the ldap DIT.
The current description states that the role grants the ldapAdminReadRole's actions with "every read action for LDAP" and on top of that, it grants permissions to "administer LDAP", I suppose this is intentionally large to encompass future administering actions.
This seems to specific:
```suggestion
	Description: "Role to allow users everything ldapAdminReadRole allows and synchronize users with an LDAP server.",
```